static/script.js

```diff
@@ -342,3 +346,13 @@ function alertIfFileContainsPassword() {
     }
     return true;
 }
+
+function alertIfFileIsEmpty() {
+    for (let child of getFiles(document.getElementById('file-container'))) {
```

Comment:
Since we now have a bit more of these checks, I think maybe it makes sense to move these `getFiles()` to the caller and save it once to reuse for all the checks.